app/assets/javascripts/discourse/app/templates/user-activity/pending.hbs

```diff
@@ -0,0 +1,39 @@
+<ul class="user-stream">
+  {{#each @model as |pending_post|}}
+    <li class="user-stream-item">
+      <div class="clearfix info">
+        <a href={{pending_post.userUrl}} data-user-card={{pending_post.username}} class="avatar-link"><div class="avatar-wrapper">{{avatar pending_post imageSize="large" extraClasses="actor" ignoreTitle="true"}}</div></a>
+        <span class="time">{{d-icon "far-clock" title="Pending"}} {{format-date pending_post.created_at}}</span>
```

Comment:
I don't remember but if format-data is just outputing text, please wrap it in a span

Reply:
So putting another `span` in the existing `span`? Like this?
```hbs
<span class="time">{{d-icon "far-clock" title="Pending"}}<span>{{format-date pending_post.created_at}}</span></span>
```

Reply:
yes, it's important to have a direct parent html element